skimage/transform/_hough_transform.pyx

```diff
@@ -314,7 +314,8 @@ def _hough_line(cnp.ndarray img,
 
 def _probabilistic_hough_line(cnp.ndarray img, int threshold,
                               int line_length, int line_gap,
-                              cnp.ndarray[ndim=1, dtype=cnp.double_t] theta):
+                              cnp.ndarray[ndim=1, dtype=cnp.double_t] theta,
+                              seed=None):
```

Comment:
I wasn't sure whether to specify a type on an optional parameter. Possibly `restoration/_unwrap_3d.pyx` shows a cythonic way to do this? There is a cdef function with `char use_seed, unsigned int seed`, which is called by a python function wrapper that sets these appropriately using it's optional seed parameter. But these hough functions are just python, so I don't know if it should be the same.

Maybe `_probabilistic_hough_line` should have required  parameters `char use_seed, unsigned int seed`, and the python wrapper `probabilistic_hough_line` sets these using the current `seed=None` optional parameter?